src/main/java/frc/robot/posev2/RobotPoseManager.java

```diff
@@ -0,0 +1,44 @@
+  
+package frc.robot.posev2;
+
+import frc.robot.RobotMap;
+
+
+public class RobotPoseManager {
+ 
+    EncoderValues encoders;
```

Comment:
these should be private. also, i would also call them something like 'lastEncoderValues'.

If you wanted to do some processing comapring the current ones with the last ones, you'd do that in the methods where you receive the new values.